### ISSUE TITLE
403 error on GravityTales

### ISSUE THREAD

**Issue by Caldar** [dteviot/WebToEpub#190]
I've tried several different stories with the same results. Here is an example:
```
Fetch of URL 'http://gravitytales.com/novel/reincarnation-of-the-strongest-sword-god/rssg-chapter-1' failed with network error 403. onResponseError@moz-extension://5ea302e3-071a-4d42-a234-addffc788b6d/js/HttpClient.js:28:25
checkResponseAndGetData@moz-extension://5ea302e3-071a-4d42-a234-addffc788b6d/js/HttpClient.js:105:20
wrapFetchImpl/<@moz-extension://5ea302e3-071a-4d42-a234-addffc788b6d/js/HttpClient.js:99:20
```

**Comment by dteviot** [comment]
@Caldar
HTTP 403 is server refusing the request.  https://en.wikipedia.org/wiki/HTTP_403
It seems to be working for me.
It might be a temporary problem on the server.  Or they've decided to block your account/IP Address for some reason.
Are you able to view the pages in the browser normally?

**Comment by Caldar** [comment]
I have no issues browsing the site normally. Not sure what the deal is but if it is working for you then I guess it must be on my end.

**Comment by dteviot** [comment]
@Caldar
Well, this is embarrassing.  Turns out there **was** a problem with WebToEpub when used in Firefox.
Has been fixed in latest version (0.0.0.61) which has been uploaded to Mozilla store.
Enjoy.
